R/eval-relocate.R

```diff
@@ -67,12 +67,18 @@ eval_relocate <- function(expr,
                           name_spec = NULL,
                           allow_rename = TRUE,
                           allow_empty = TRUE,
+                          allow_predicates = NA,
```

Comment:
I was actually not under the impression that this would be a user facing argument. I thought the generics would be in full control of whether or not predicates are allowed.

Reply:
Yeah, that was the original goal, but then it seemed inconsistent to not also have it at the top level.